shell/app/modules/project/pages/release/components/application-detail.tsx

```diff
@@ -108,13 +131,26 @@ const ReleaseApplicationDetail = ({ isEdit = false }: { isEdit: boolean }) => {
                     label={i18n.t('dop:version name')}
                     name="version"
                     type="input"
+                    formItemProps={{
+                      validateStatus: !isUnique ? 'error' : undefined,
+                      help: !isUnique
+                        ? i18n.t('{name} already exists', { name: i18n.t('dop:version name') })
+                        : undefined,
+                    }}
                     rules={[
                       { required: true, message: i18n.t('please enter {name}', { name: i18n.t('dop:version name') }) },
                       { max: 30, message: i18n.t('dop:no more than 30 characters') },
                       {
                         pattern: /^[A-Za-z0-9._-]+$/,
                         message: i18n.t('dop:Must be composed of letters, numbers, underscores, hyphens and dots.'),
                       },
+                      {
+                        validator: (_, value: string) => {
+                          setIsUnique(true);
+                          check(value);
+                          return Promise.resolve();
```

Comment:
I guess the `validateStatus` should be controlled by this promise, but here always return reslove